### PR TITLE
Corrected Linux console keyboard mapping for Portuguese (Brazil), Czech, Slovak

### PR DESCRIPTION
**Pull request by ancorgs**
Obsoletes #84 

Includes some reformatting of the file for legibility.

@embar-, @marceloatie, could you take a look and comment if you are fine with merging this as a replacement of #84?

Do not merge before checking Jenkins status first (I suspect is wrongly configured regarding branches).

### CODE REVIEW

**Comment by embar-** [comment]
Nice! 

Now I see, that how 9cac07d commit (Remove artifact in Portuguese (Brazil -US) config) introduced syntax error:

For long time there was incorrect line:
` "sparc64"	: $[ "type4": $[ "ncurses": "us.map.gz"],`

But at end of Portuguese section was also additional "]", so this not resulted in syntax error.
After correction of line, it was also necessary to remove this additional "]", as AncorGS did in 28d65fc.

Lets include this patch!

File: keyboard/src/data/keyboard_raw.ycp [hunk 1]
```diff
@@ -230,13 +230,13 @@ return ($[
 	$[
 	    "pc104"	: $[ "ncurses": "us-intl.map.gz"],
 	    "macintosh" : $[ "ncurses": "us-mac.map.gz"],
-	    "sparc64"	: $[ "type4": $[ "ncurses": "us.map.gz"],
+	    "type4"	: $[ "ncurses": "us.map.gz"],
 	    "type5"	: $[ "ncurses": "us.map.gz" ],
 	    "type5_euro": $[ "ncurses": "us.map.gz" ],
 	]
     ],
-  ],
```
Comment (lslezak) [inline keyboard/src/data/keyboard_raw.ycp]: Oh, at first sight it looked like a mistake to remove this, but it was actually part of the unclosed `sparc64` key, which was a bug I guess...

**Comment by lslezak** [comment]
JFYI: I did a quick syntax check of the `keyboard_raw.ycp` (using ` ycpc -E -q keyboard_raw.ycp`) and it is OK.

I'm not a keyboard expert, but code-wise it LGTM.

**Comment by jreidinger** [comment]
also for me it looks fine

**Pull request merged**